docs/developers-clear-containers-install.md

```diff
@@ -5,7 +5,23 @@ instructions for any developers eager to try Clear Containers 3.0 and who
 want to build Clear Containers from the source code and are familiar with the
 process.
 
-## Requirements
+## Initial setup
+
+The recommended way to create a development environment is to first install the
+packaged versions of the Clear Containers components to create a working
+system:
+
+  * [Fedora](fedora-installation-guide.md)
+  * [Ubuntu](ubuntu-installation-guide.md)
+
+The instructions will install all packaged components, plus docker,
```

Comment:
state which instructions you are referring to:
The installation guide instructions OR The instructions in this guide